src/helpers.rs

```diff
@@ -480,6 +482,25 @@ pub trait EvalContextExt<'mir, 'tcx: 'mir>: crate::MiriEvalContextExt<'mir, 'tcx
         self.eval_context_mut().memory.write_bytes(scalar, bytes.iter().copied().chain(iter::once(0u8)))?;
         Ok(true)
     }
+
+    // Converts an operand with a machine `isize` into the desired integer type using `TryInto`.
+    // This function is useful when reading syscall related operands which are delivered as pointer
+    // sized operands.
+    fn try_from_ptr_sized_operand<T: TryFrom<i64, Error=TryFromIntError>>(
```

Comment:
Given that this only has two sues, I am not sure this carries its weight... I can hardly parse the signature of this function.^^

Reply:
heh ok